src/bot/avalanche.ts

```diff
@@ -1,27 +1,38 @@
 import { BaseBot, Bot } from './bot.js';
 import { Status } from '../service/mastodon.js';
-import { getAvalancheReport } from '../service/avalanche/avalanche.js';
-import humanDate from 'human-date';
+import {
+  AvalancheReport,
+  getAvalancheReport,
+} from '../service/avalanche/avalanche.js';
+import dateFnsTz from 'date-fns-tz';
+const { formatInTimeZone } = dateFnsTz;
+
+const STATUS_TIME_ZONE = 'America/Vancouver';
 
 export default class AvalancheReportBot extends BaseBot implements Bot {
   async postStatus(): Promise<Status | undefined> {
     const report = await getAvalancheReport();
     if (!report) {
       return;
     }
-    const statusText = `Forecast for Kootenay Boundary issued at ${humanDate.prettyPrint(
-      report.dateIssued,
-      { showTime: true }
-    )} UTC:
+    console.log(report);
```

Comment:
I'm surprised eslint didn't pick up these console logs, but I think they're useful for debugging for now anyway

Reply:
oops, yes i think we may need to add a no-console rule to eslint, it may not be default.